src/@ionic-native/plugins/purchases/index.ts

```diff
@@ -682,7 +737,7 @@ export class Purchases extends IonicNativePlugin {
    * @param onesignalID Empty String or null will delete the subscriber attribute.
    */
   @Cordova({ sync: true })
-  setOnesignalId(onesignalId: string | null): void {}
+  setOnesignalID(onesignalID: string | null): void {}
```

Comment:
These changes are going to break people's implementations. Did we change the names in cordova too?

Reply:
actually it looks like they've always been uppercase in cordova, so not sure how this ever worked...

see: https://github.com/RevenueCat/cordova-plugin-purchases/commit/c803df61a861b6e96b3ac24b465bfde80a106baf#diff-5776869548f9a7e95693a9bf21febcb3d6e3c808fc9ed4533444e03781400958